packages/protocol/utils/generate_genesis/taikoL2.ts

```diff
@@ -392,8 +415,11 @@ async function generateContractConfigs(
         },
         BridgedERC20: {
             address: addressMap.BridgedERC20Impl,
-            deployedBytecode:
-                contractArtifacts.BridgedERC20Impl.deployedBytecode.object,
+            deployedBytecode: replaceUUPSImmutableVaules(
+                contractArtifacts.BridgedERC20Impl,
+                uupsImmutableReferencesMap,
+                ethers.utils.hexZeroPad(addressMap.BridgedERC20Impl, 32),
+            ).deployedBytecode.object,
         },
         BridgedERC721: {
```

Comment:
I see `BridgedERC1155` and `BridgedERC20` getting patched, but not ` BridgedERC721`. Shouldn't it also be patched?

Reply:
Plz review: https://github.com/taikoxyz/taiko-mono/pull/15479